src/wasm/rlottiewasm.cpp

```diff
@@ -41,6 +41,157 @@ class __attribute__((visibility("default"))) RlottieWasm {
 
         return val(typed_memory_view(mWidth * mHeight * 4, mBuffer.get()));
     }
+
+    void setFillColor(std::string key, float r, float g, float b){
+        mPlayer->setValue<rlottie::Property::FillColor>(key,
+                [r,g,b](const rlottie::FrameInfo& info){
+                    return rlottie::Color(r, g, b);
+                });
+    }
+  
+    void setFillOpacity(std::string key, float opacity) {
+        mPlayer->setValue<rlottie::Property::FillOpacity>(key,
+                [opacity](const rlottie::FrameInfo& info) {
+                    return opacity;
+                });
+    }
+
+    void setStrokeColor(std::string key, float r, float g, float b){
+        mPlayer->setValue<rlottie::Property::StrokeColor>(key,
+                [r,g,b](const rlottie::FrameInfo& info){
+                    return rlottie::Color(r, g, b);
+                });
+    }
+
+    void setStrokeOpacity(std::string key, float opacity) {
+        mPlayer->setValue<rlottie::Property::StrokeOpacity>(key,
+                [opacity](const rlottie::FrameInfo& info) {
+                    return opacity;
+        });
+    }
+
+    void setStrokeWidth(std::string key, float width) {
+        mPlayer->setValue<rlottie::Property::StrokeWidth>(key,
+                [width](const rlottie::FrameInfo& info) {
+                    return width;
+                });
+    }
+
+    void setTrAnchor(std::string key, float x, float y){
```

Comment:
no need to pass a lambda when we have api which takes a static value. Same for below functions as well.
mPlayer->setValue<rlottie::Property::StrokeWidth>(key, width) will do the job.

Reply:
Oh, Thank you. 
I'll modify the code.